src/s1ap/handlers/ctxrelease_resp.c

```diff
@@ -36,22 +36,16 @@ extern int g_enb_fd;
 extern ipc_handle ipcHndl_ctx_release_complete;
 
 int
-s1_ctx_release_resp_handler(char *msg)
+s1_ctx_release_resp_handler(InitiatingMessage_t *msg)
 {
 	struct ctx_release_complete_Q_msg release_complete;
 	struct proto_IE s1_ctx_release_ies;
 
-	unsigned short msg_len = get_length(&msg);
-
-	char *buffer;
-	log_msg(LOG_INFO, "S1AP_UE_CTX_RELEASE msg: %s\n", msg_to_hex_str(msg, msg_len, &buffer));
-	free(buffer);
```

Comment:
Did we removed this logging ?

Reply:
yes. Now the decoder already gives us a structure variable. So this character buffer is not available anymore.